package.json

```diff
@@ -25,6 +25,9 @@
     "build-languages": "gulp languages",
     "docker:compose": "docker-compose -f docker/docker-compose.yml",
     "docker:up": "yarn docker:compose up",
+    "docker:up:php7.2": "DOCKER_PHP_VERSION=7.2-apache yarn docker:compose up",
+    "docker:up:php7.1": "DOCKER_PHP_VERSION=7.1-apache yarn docker:compose up",
+    "docker:up:php7.0": "DOCKER_PHP_VERSION=7.0-apache yarn docker:compose up",
```

Comment:
Shall we add v5 here? https://hub.docker.com/_/php/

I was exploring if it would be possible to do `yarn docker:up --php=7.1` or something similar but I didn't really find a way to grab those args in a way that would work windows/linux/mac and wouldn't require adding extra node/bash scripts.

Reply:
Yeah we should. It's just that some build step was throwing an error for me with 5.6 and I was lazy to solve it at the moment. 

Just wanted to get the flow of multi php working with this image and prove if it sucked or not to switch versions in this way. Not still quite sure if it's comfortable or not. It did come in handy to have it since I pushed this changes.

Reply:
Yeah, developer experience definitely is a big part of this so I wouldn't rush this in just when it technically already works.

Reply:
indeed, neither would I.